lake/definitions/aha_config_pairs.json

```diff
@@ -1,29 +1,31 @@
 {
-  "seeds": 10,
+  "seeds": 2,
+  "train_epochs": 5,
   "experiment_name": "associative_inference",
-  "learning_type": "static",
+  "learning_type": "recurrence",
   "communities": 3,
   "test_single_characters": true,
   "test_mirror_characters": true,
+  "activation": true,
 
   "characters": 9,
-  "sequence_length": 600,
-  "study_batch_size": 60,
-  "settled_response_steps": 50,
+  "sequence_length": 60,
+  "study_batch_size": 20,
+  "settled_response_steps": 100,
   "initial_response_step": 20,
 
   "alphabet": "Latin",
   "variation": false,
   "character_idx_study": 1,
   "character_idx_recall": 1,
 
-  "pretrain_epochs": 5,
+  "pretrain_epochs": 10,
   "pretrain_batch_size": 150,
   "image_resize_factor": 0.5,
   "image_shape": [1, 1, 52, 104],
 
 
-  "test_stm_components": ["dg", "ca3", "pr", "pm_ec", "final"],
+  "test_stm_components": ["dg", "ca3", "pr", "ca3_ca1", "final"],
```

Comment:
Should also add CA1; the CA1 is the one we expect (as per Schapiro) to be the star for these experiments.. somehow :D